unit-test/background/reference-tests/3p-cookies-tests.js

```diff
@@ -7,7 +7,7 @@ const tdsStorage = require('../../../shared/js/background/storage/tds').default
 const tabManager = require('../../../shared/js/background/tab-manager')
 const browserWrapper = require('../../../shared/js/background/wrapper')
 const { dropTracking3pCookiesFromResponse, dropTracking3pCookiesFromRequest } = require('../../../shared/js/background/events/3p-tracking-cookie-blocking')
-const getArgumentsObject = require('../../../shared/js/background/helpers/arguments-object')
+const { getArgumentsObject } = require('../../../shared/js/background/helpers/arguments-object')
```

Comment:
For unit-tests, `getUserLocale` will fall back to `en`, as the `browser.i18n` API is missing in the test context.

For integration tests, I think Playwright runs an `en-US` locale by default (this can be [overridden](https://playwright.dev/docs/emulation#locale--timezone) in the test config). 

Therefore, I don't think this would be an issue.